README.md

```diff
@@ -162,30 +154,24 @@ First get the saved query id to run with `get savedqueries` command, then use th
 
     
 
-**Log Nicknames, Log Groups, and Query Nicknames**
+**CLI favorites and Log Sets**
 --------------------------------------------------
-The CLI supports the use of log nicknames and log groups via the configuration file. This makes searching well known or large lists of logs much simpler as you do not need to pass in lists of log Ids.
-
-####Log Nicknames
-Log nicknames allow an alias for a single log to be configured, this is done in the LogNicknames section of the configuration file. 
-```
-[LogNicknames]
-testlog = 12345678-aaaa-bbbb-1234-1234cb123456
-```
+The CLI supports the use in query commands of CLI Favorites (log alias) via the configuration file and log sets from the Logentries account. This makes searching well known or large lists of logs much simpler as you do not need to pass in lists of log Ids.
```

Comment:
It would probably be good to start with something like lecli supports command line favorites (CLI favorites) for query commands.

Reply:
going to modify this slightly